locales/zh_CN/LC_MESSAGES/offset.po

```diff
@@ -76,4 +90,17 @@ msgid ""
 "g = ti.var(dt=ti.i32, shape=(16, 32), offset=-16)                    # "
 "AssertionError"
```

Comment:
... but `AssertionError` here?

Reply:
Maybe it is more like a **TypeError** here ?

Reply:
I think it may be likely a typo in en doc...

Reply:
> I think it may be likely a typo in en doc...

> Maybe it is more like a **TypeError** here ?

Just now I tested these illegal expressions in my end. `AssertionError` but not `TypeError` will be thrown out, the doc matches the behavior.

Another question is that, Is that what you think is inappropriate?